src/engine/Camera.ts

```diff
@@ -126,23 +141,29 @@ module ex {
       }
 
       /**
-       * Sets the focal point of the camera. This value can only be set if there is no actor to be followed.
-       * @param x The x coordinate of the focal point
-       * @param y The y coordinate of the focal point
-       * @deprecated
+       * This moves the camera focal point to the specified position using specified easing function. Cannot move when following an Actor.
+       * 
+       * @param pos The target position to move to
+       * @param duration The duration in millseconds the move should last
+       * @param [easingFn] An optional easing function ([[ex.EasingFunctions.EaseInOutCubic]] by default)
        */
-      public setFocus(x: number, y: number) {
-         if (!this._follow && !this.lerp) {
-            this.x = x;
-            this.y = y;
+      public move(pos: Vector, duration: number, easingFn: EasingFunction = EasingFunctions.EaseInOutCubic) {
```

Comment:
It would useful if `move` returned a promise that resolved when this movement is complete